packages/block-library/src/archives/edit.js

```diff
@@ -36,7 +40,7 @@ export default function ArchivesEdit( { attributes, setAttributes } ) {
 			<Disabled>
 				<ServerSideRender
 					block="core/archives"
-					attributes={ attributes }
+					attributes={ omit( attributes, [ 'align' ] ) }
```

Comment:
What's the reason behind this change?

Reply:
Thanks for dropping in so quickly!

This is probably not the best solution, but it seems that removing the `align` attribute from block.json causes the SSR components request to 400:

![Screen Shot 2020-07-28 at 12 07 02 PM](https://user-images.githubusercontent.com/28742426/88691562-0049da80-d0cb-11ea-96e4-2f15f7d65d47.png)

Since align is a saved attribute (but no longer in block.json?) it seems to be seen as an invalid attribute.  Breaking the block:

![Screen Shot 2020-07-28 at 12 09 57 PM](https://user-images.githubusercontent.com/28742426/88691865-65053500-d0cb-11ea-93c2-e27e0c3818d7.png)

If we omit the align attribute, the block continues to work as expected.  I think using `omit` is a bit hacky, so maybe we don't want to do it this way.  Perhaps this is why the attributes weren't removed previously? 🤔

Reply:
Where's the "align" attribute registration happening now. It seems that the server-side registration is not picking up that attribute definition?

Reply:
I guess what I'm saying is that the `addAttribute` filter needs to move to the server now. We probably need to keep in both places though.

Reply:
I'm pretty sure the other attributes registered using "hooks" (color, lineHeight...) suffer the same issue?

Reply:
> I guess what I'm saying is that the addAttribute filter needs to move to the server now. We probably need to keep in both places though.

Yeah, that makes sense.  I wonder where this should be handled?  

> I'm pretty sure the other attributes registered using "hooks" (color, lineHeight...) suffer the same issue?

I would assume so, but I haven't run across any similar issues with them yet.

Im thinking it might make sense to do this in a future PR as it may inflate a bit from the 'simple cleanup' we were going for here.  While #24122 allows us to clean up some stuff in the index.php's here, this blocker for removing the defined align attribute for blocks using the SSR component seems to be a separate issue that will need to be addressed for all of the hooks.  So for these blocks I assume leaving their attributes as they were would be preferable to using the `omit` function?

Reply:
Updated to not remove the align attribute for blocks currently using the SSR component (no more `omit`, and their cleanup is limited to their index.php).  These blocks are:
* Archives
* Calendar
* Latest Comments
* RSS
* Tag Cloud

The following blocks do not use the SSR component and seem to have no issue with their align attribute being removed:
* Categories
* Latest Posts
* Navigation
* Search